cloudbuild.yaml

```diff
@@ -162,27 +98,6 @@ steps:
     entrypoint: 'bash'
     args: ['./scripts/build_examples_android']
 
-  - name: 'gcr.io/oak-ci/oak:latest'
-    id: check_formatting
-    waitFor: ['git_init', 'build_android_image']
-    timeout: 30m
-    entrypoint: 'bash'
-    args: ['./scripts/check_formatting']
```

Comment:
Does `runner` do the check that all BUILD files have a `licenses = ["notice"]` directive?

Reply:
Yes now, thanks!